ChatterboxBackend/Chatterbox/Models/Chat.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations.Schema;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Chatterbox.Models
+{
+    public class Chat

```

Comment:
Is Chat still being used? Its `DbSet` was removed from the context. Remove class if unused.